src/screens/FeedScreen/RenderList.js

```diff
@@ -231,7 +332,8 @@ RenderListFeed.propTypes = {
   source: PropTypes.string,
   hideThreeDot: PropTypes.bool,
   showAnonymousOption: PropTypes.bool,
-  onHeaderOptionClicked: PropTypes.func
+  onDeletePost: PropTypes.func,
+  isSelf: PropTypes.bool
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

New PropTypes have been defined. Make sure they match the data types of the actual props passed to the component.

```diff
- onHeaderOptionClicked: PropTypes.func

+ onDeletePost: PropTypes.func,
+ isSelf: PropTypes.bool
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->